nixos/modules/services/web-apps/languagetool.nix

```diff
@@ -0,0 +1,47 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  cfg = config.services.languageToolHttp;
+  format = pkgs.formats.toml { };
+in
+{
+  options.services.languageToolHttp = {
+    enable = mkEnableOption "the LanguagTool http server";
+    allowBrowserPluginAccess = mkEnableOption "access from the browser plugin";
+    settings = mkOption {
+      description = ''
+        Contents of the configuration file.
```

Comment:
Please link to specification. After a very quick look at the github project I didn't see anything about it.